app/ui/src/main/res/layout/split_message_list.xml

```diff
@@ -8,6 +8,18 @@
 
     <include layout="@layout/toolbar" />
 
+    <ProgressBar
```

Comment:
Does this fit into `@layout/toolbar` for less code duplication?

Reply:
Hm, I hadn't considered this. For now the progress bar is only used in the message list screen. So I wouldn't want to add it to all toolbars. I might create a `toolbar_message_list` layout in the future. But for now I think I'll keep it as is.